spark/src/test/scala/org/apache/comet/CometExpressionSuite.scala

```diff
@@ -2529,4 +2529,21 @@ class CometExpressionSuite extends CometTestBase with AdaptiveSparkPlanHelper {
         spark.sql("SELECT array_contains((CASE WHEN _2 =_3 THEN array(_4) END), _4) FROM t1"));
     }
   }
+  test("array_remove") {
+    Seq(true, false).foreach { dictionaryEnabled =>
+      withTempDir { dir =>
+        val path = new Path(dir.toURI.toString, "test.parquet")
+        makeParquetFileAllTypes(path, dictionaryEnabled, 10000)
+        spark.read.parquet(path.toString).createOrReplaceTempView("t1");
```

Comment:
nit: no need for semicolon:

```suggestion
        spark.read.parquet(path.toString).createOrReplaceTempView("t1")
```